login-workflow/src/components/SetPassword/types.ts

```diff
@@ -71,4 +72,10 @@ export type SetPasswordProps = {
      * @returns void
      */
     onSubmit?: () => void;
+
```

Comment:
you can refer to account details screen props for this but we've been using this for docs. But update everywhere where TextFieldProps will be used

/**
         * The props to pass to the password field.
         * See [MUI's TextFieldProps API](https://mui.com/material-ui/api/text-field/) for more details.
         */
        passwordTextFieldProps?: TextFieldProps;